samigo/samigo-app/src/webapp/jsf/delivery/item/deliverFillInNumeric.jsp

```diff
@@ -25,7 +25,8 @@ should be included in file importing DeliveryMessages
 --%>
 -->
 <script>includeWebjarLibrary("qtip2");</script>
-<script>includeWebjarLibrary("bootstrap");</script>
+<script src="/library/webjars/bootstrap/3.3.7/js/bootstrap.min.js<h:outputText value="#{delivery.CDNQuery}" />"></script>

```

Comment:
This worries me a little bit. includeWebjarLibrary should build the exact same script tag. I presume there was a reason you had to do this? Could you explain, please?

Reply:
You can fix the popover error by switching it to bootstrap.Popover. Take a look at the bootstrap transition google doc

Reply:
Hi Adrian, I added the includeWebjarLibrary in order to solve the popover issue in the Lessons tool ([SAK-47188](https://sakaiproject.atlassian.net/browse/SAK-47188)). But I guess it would be better to address the popover issue elsewhere and remove the bootstrap library from this code.

Reply:
Hi Adrian, I removed the code: ```<script src="/library/webjars/bootstrap/3.3.7/js/bootstrap.min.js<h:outputText value="#{delivery.CDNQuery}" />"></script>```. Would you like to take a look at it?